lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -598,54 +598,43 @@ function writeStringsToCtcFiles(locale, strings) {
 }
 
 /**
- * This function does three things:
+ * This function does two things:
```

Comment:
sorry @patrickhulce, this would have been nice to know before your changes. And I've bungled up your nice cascade of checks a bit :)

Reply:
Haha, no worries. It's still more straightforward to read IMO so still a win :)